packages/components/ContextualMenu/src/SubmenuItem.settings.ts

```diff
@@ -15,26 +15,35 @@ export const settings: IComposeSettings<SubmenuItemType> = [
       focusable: true,
       style: {
         display: 'flex',
-        alignItems: 'flex-start',
         flexDirection: 'row',
         alignSelf: 'flex-start',
         width: '100%',
+        justifyContent: 'space-between',
       },
     },
     content: {},
-    icon: {},
-    stack: {
+    icon: {style: { marginEnd: 5 }},
+    leftstack: {
       style: {
         display: 'flex',
-        paddingStart: 16,
-        paddingEnd: 16,
+        paddingStart: 5,
         alignItems: 'center',
         flexDirection: 'row',
         alignSelf: 'flex-start',
         minHeight: 32,
-        minWidth: 80,
-        justifyContent: 'center',
-      },
+        justifyContent: 'flex-start',
+      }
+    },
+    rightstack: {
+      style: {
+        display: 'flex',
+        paddingEnd: 5,
```

Comment:
Does the UI look consistent across all DPIs?

Reply:
I believe so!